config/app.php

```diff
@@ -78,7 +78,17 @@
     |
     */
 
-    'locale' => 'en',
+    //'locale' => 'en',
```

Comment:
You should not change locale to 'fr', as this will change it for everyone.

Reply:
I think a good idea here is to make the value an environment variable, on this way the people who wants to use the project don't need to touch this file, only the .env